### DISCUSSION TITLE
Can we raise funds for a new driver?

### DISCUSSION THREAD

**Discussion by zxcvqwerasdf** [Rem0o/FanControl.Releases#1982] [Ideas]
For now FanControl using old WinRing0.sys
![image](https://github.com/Rem0o/FanControl.Releases/assets/67195196/2212145b-f1a2-415a-9571-c70755ddc928)
It's very old version with bugs and vulnerabilities.
Maybe we can raise funds for EV Certificate to sign new driver?

**Comment by RainerRocks** [comment]
How much does it cost ?

**Comment by zxcvqwerasdf** [comment]
> How much does it cost ?

For example:
https://www.digicert.com/signing/code-signing-certificates
~629$ for 1 year (but even after expires it will work, same as current driver signature (what expired ‎9 september ‎2008) )

**Reply by Rem0o** [reply]
Rules changed for driver certifications. The type of signing that was used for WinRing0 no longer works. You can read about the new rules here: https://learn.microsoft.com/en-us/windows-hardware/drivers/dashboard/hardware-program-register
